frontend/packages/ux-editor/src/containers/FormDesigner.tsx

```diff
@@ -142,11 +147,41 @@ export const FormDesigner = (): JSX.Element => {
     return (
       <DragAndDropTree.Provider rootId={BASE_CONTAINER_ID} onMove={moveItem} onAdd={addItem}>
         <div className={classes.root}>
+          <FormDesignerToolbar></FormDesignerToolbar>
           <div className={classes.container}>
-            <Elements />
-            <DesignView />
-            <Properties />
-            <Preview />
+            <StudioResizableLayout.Container
+              orientation='horizontal'
+              localStorageContext={`form-designer-main:${user.id}:${org}`}
+            >
+              <StudioResizableLayout.Element
+                collapsed={elementsCollapsed}
+                collapsedSize={49}
+                minimumSize={262}
+                maximumSize={300}
+              >
+                <Elements
+                  collapsed={elementsCollapsed}
+                  onCollapseToggle={() => setElementsCollapsed((prev) => !prev)}
+                />
+              </StudioResizableLayout.Element>
+              <StudioResizableLayout.Element>
+                <DesignView />
+              </StudioResizableLayout.Element>
+              <StudioResizableLayout.Element onResizing={(resizing) => setHidePreview(resizing)}>
+                <Properties />
+              </StudioResizableLayout.Element>
+              <StudioResizableLayout.Element
+                collapsed={previewCollapsed}
+                collapsedSize={49}
+                minimumSize={400}
+              >
+                <Preview
+                  collapsed={previewCollapsed}
+                  onCollapseToggle={() => setPreviewCollapsed((prev) => !prev)}
```

Comment:
Is it possible to write a test that covers this?

Reply:
Yes, good point 👍 done ✅